### PR TITLE
counting scanned lines in packing menu

### PR DESCRIPTION
**Pull request by waf31**
Request from Oliverstore

### CODE REVIEW

**Comment by sylvainmerlin** [comment]
@hparfr can you review it ?

File: justin/src/colisage/colisage.html [hunk 1]
```diff
@@ -5,7 +5,7 @@
       </ion-buttons>
 
       <ion-title>
-        Packing
+        Packing, items scanned {{productCount}}
```
Comment (hparfr) [inline justin/src/colisage/colisage.html]: `{{ model.products.length }}`

File: justin/src/colisage/colisage.page.ts [hunk 2]
```diff
@@ -15,6 +15,7 @@ import {Pack, Shipment, Product } from '../statemachine/src/states';
 export class ColisagePage {
   shipment: any;
   model: any = {};
+  productCount: number = 0;
```
Comment (hparfr) [inline justin/src/colisage/colisage.page.ts]: not needed

File: justin/src/colisage/colisage.page.ts [hunk 3]
```diff
@@ -43,6 +44,7 @@ export class ColisagePage {
     return toast.present();
   }
   addIt(scanned) {
+    this.productCount ++;
```
Comment (hparfr) [inline justin/src/colisage/colisage.page.ts]: not needed

File: justin/src/colisage/colisage.page.ts [hunk 4]
```diff
@@ -102,6 +104,7 @@ export class ColisagePage {
   }
 
   reset(withShipment) {
+    this.productCount = 0;
```
Comment (hparfr) [inline justin/src/colisage/colisage.page.ts]: not needed

**Review by hparfr — CHANGES_REQUESTED** [review]
Welcome on this project and thanks for this first PR.

You don't need to create and manage a new variable. You can simply query the lengh of the array directly in the view.

**Comment by sylvainmerlin** [comment]
Thanks !

**Comment by waf31** [comment]
Hi @hparfr, i have followed your tip and rollback my changes, instead of this i have put your idea of displaying model.products.length

**Review by hparfr — APPROVED** [review]

**Pull request merged**